### DISCUSSION TITLE
Can I specifiy first/last edge segment direction using elk.port.side

### DISCUSSION THREAD

**Discussion by czc31** [eclipse-glsp/glsp#1271] [Q&A]
In another use of sprotty and elk it was possible to define the first and the last segment orientation according to a port property elk.port.side. Something like if the port is on the left side of a node then the edge connected to this edge wil start by following a west direction. Is this kind of feature is feasible with GLSP and how ?
Many thanks in advance

**Comment by tortmayr** [comment]
Hi @czc31 
sorry for the late reply. Somehow your question seems to have fallen through the cracks.
Yes this is possible. Similar to sprotty GLSP provides an integration with elk. 
Please let me know in which server flavor you are interested in (java or node/ts) so that I can provide you with more concrete steps.

**Reply by david-pa** [reply]
Is there any info available on how to do this with a node/ts server flavor? I'm wondering if this would potentially be a solution to my own issue. https://github.com/eclipse-glsp/glsp/discussions/1501
